### PR TITLE
Add data persistence to ibutsu-pod.sh script

### PR DESCRIPTION
**Pull request by rsnyman**
- Add ability to specify options in ibutsu-pod.sh
- Detect if script is running in the right directory
- Add options to the script for creating admin and project
- Add option to enable data persistence in postgres and redis

### CODE REVIEW

**Comment by rsnyman** [comment]
@Griffin-Sullivan take a look at this

File: scripts/ibutsu-pod.sh [hunk 1]
```diff
@@ -12,13 +82,15 @@ podman run -dt \
        -e POSTGRES_USER=ibutsu \
        -e POSTGRES_DB=ibutsu \
        -e POSTGRES_PASSWORD=ibutsu \
+       $POSTGRES_EXTRA_ARGS \
```
Comment (Griffin-Sullivan) [inline scripts/ibutsu-pod.sh:85]: At https://hub.docker.com/_/postgres they also recommend storing the data under another folder in `/var/lib/postgres/data`. See the section titled `PGDATA`. We could easily add that with this var.

File: scripts/ibutsu-pod.sh [hunk 2]
```diff
@@ -1,7 +1,77 @@
...
+# Persist the data
+if [[ $IS_PERSISTENT = true ]]; then
+    POSTGRES_EXTRA_ARGS+=' -v ./.postgres-data:/var/lib/postgres/data:Z'
+    REDIS_EXTRA_ARGS+=' -v ./.redis-data:/data:Z'
+    if [[ ! -d ".postgres-data" ]]; then
+        mkdir .postgres-data
+    fi
+    if [[ ! -d ".redis-data" ]]; then
+        mkdir .redis-data
+    fi
```
Comment (Griffin-Sullivan) [inline scripts/ibutsu-pod.sh:66]: Could also make these volumes with the `podman volume create` command and use them in vars above.
```bash
if volumes don't exist:
    podman volume create postgres-data
    podman volume create redis-data
POSTGRES_EXTRA_ARGS+=' -v postgres-data:/var/lib/postgres/data:Z'
REDIS_EXTRA_ARGS+=' -v redis-data:/data:Z'
```
Sorry I've never put in the time to understand bash if statements....

Comment (rsnyman) [reply scripts/ibutsu-pod.sh:66]: I've never been a huge fan of volumes. They're a bit opaque, and can sometimes cause issues when you don't realise that there's a volume with the same name somewhere else.

File: scripts/ibutsu-pod.sh (hunk 1 above)
Comment (rsnyman) [reply scripts/ibutsu-pod.sh:105]: OK, I've specified the PGDATA variable.

File: scripts/ibutsu-pod.sh (hunk 2 above)
Comment (rsnyman) [reply scripts/ibutsu-pod.sh]: I added an option to use volumes.

**Comment by rsnyman** [comment]
The checks are failing due to external issues, and these issues are already solved in #313, so I've marked it as a draft.

**Pull request merged**